src/components/Create/components/RewardsList/RewardItem.tsx

```diff
@@ -101,7 +105,7 @@ export const RewardItem = ({
               title={t`Minimum contribution`}
               stat={`${numberUpToPrecisionFormat(contributionFloor)} ETH`}
             />
-            {!!maxSupply && (
+            {!!maxSupply && maxSupply !== DEFAULT_NFT_MAX_SUPPLY && (
```

Comment:
**suggestion** perhaps we should have a single util for hasUnlimitedSpply or something